README.md

```diff
@@ -127,7 +127,7 @@ The following optional parameters are available:
 
 As for the options, the following are available:
 
--   `customForceUpdateHook`: pass a custom hook that should make a component re-render (or not) when changes are detected.
+-   `useForceUpdate`: pass a custom hook that should make a component re-render (or not) when changes are detected.
```

Comment:
Some mention about default should be here. It sounds like you need to specify this every time :)

Reply:
done